apps/docs/content/guides/functions/limits.mdx

```diff
@@ -18,6 +18,7 @@ subtitle: "Limits applied Edge Functions in Supabase's hosted platform."
 ## Other limits & restrictions
 
 - Outgoing connections to ports `25` and `587` are not allowed.
-- Serving of HTML content is not supported (`GET` requests that return `text/html` will be rewritten to `text/plain`).
+- Serving of HTML content is only supported with [custom-domains](https://supabase.com/docs/reference/cli/supabase-domains) ( Otherwise `GET` requests that return `text/html` will be rewritten to `text/plain`).
```

Comment:
```suggestion
- Serving of HTML content is only supported with [custom domains](/docs/reference/cli/supabase-domains) (Otherwise `GET` requests that return `text/html` will be rewritten to `text/plain`).
```